### ISSUE TITLE
In the netlogo code, father removal is partial

### ISSUE THREAD

**Issue by mariopaolucci** [LABSS/PROTON-OC#197]
When we remove fathers, we cut **only** the *offspring* links.

That means we keep other links, of which household is most important; and since household links are included in person links, that father can potentially still commit crimes with that offspring.

Is this correct? Would it make a difference if we introduce a stronger removal?

**Comment by Kubasinska** [comment]
Correct, and I would add that this also creates problems in the process of returning children.
